src/card-stack/card-stack.html

```diff
@@ -62,6 +62,17 @@
         // static properties on the GameCard class.
         return GameCard[suit.toUpperCase()];
       }
+
+      _dragOver(e){
+        e.preventDefault();
+      }
+      _drop(e){
+        e.preventDefault();
+        var card = e.dataTransfer.getData('card');
+        var stack = e.target;
+        this.dispatchEvent(new CustomEvent('card-dropped', {detail:{card: card, stack:stack}}));
```

Comment:
Also, as a cool trick in modern JavaScript, if you have a variable and you want to add it to an object literal with its name as its key you can just put the name of the variable alone.

So for example, the following are identical: `{card: card, stack: stack}` and `{card, stack}`.